### PR TITLE
Create ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59454](https://cumulocity.atlassian.net/browse/MTM-59454)
Original PR: [6589](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6589)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: add permissions check for the copy dashboard action (#6493) [GRAFT][release/cd] (#6589)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add permissions check for copy dashboard action
```

File: content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59454
+version: 1020.2.5
+---
+add permissions check for the copy dashboard action (#6493) [GRAFT][release/cd] (#6589)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
To enhance security and access control, a permissions check has been added when copying dashboards. Previously, any user could copy a dashboard regardless of their permissions. With this change, only users with the necessary permissions to create a new dashboard are allowed to perform the copy action. This ensures that users cannot bypass access restrictions by copying dashboards they are not authorized to create from scratch.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by rahultsag — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 date: ""
-title: add permissions check for the copy dashboard action (#6493) [GRAFT][release/cd] (#6589)
+title: Add permissions check for copy dashboard action
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md:3]:
```suggestion
title: Added a permission check for the copy dashboard action
```

File: content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59454
+version: 1020.2.5
+---
+Added a permissions check for the copy dashboard action. The system will now verify that the user has the necessary permissions before allowing them to copy a dashboard. If the user lacks the necessary permissions, the copy button will be disabled.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-2-5-add-permissions-check-for-the-copy-dashboard-action-6493-graft-release.md:17]:
```suggestion
A permission check for the copy dashboard action has been added. The platform now verifies that the user has the required permissions before allowing them to copy a dashboard. If the user lacks the required permissions the **Copy** button will be disabled.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**